tests/unit/internal-session-test.js

```diff
@@ -385,7 +385,17 @@ describe('InternalSession', () => {
       return session.authenticate('authenticator');
     });
 
-    describe('when the authenticator resolves invaldiation', () => {
+    describe('when the authenticator resolves invalidation with params', () => {
+      it('when some data is sent with invalidate', () => {
+        let invalidateSession = sinon.spy(authenticator, 'invalidate');
+        let param = { some: 'random data' };
+        session.invalidate(param);
+        invalidateSession.restore();
```

Comment:
You'll always want to set up spies (and especially restore the original methods) in `beforeEach`/`afterEach` hooks so you can be sure spies will always be cleaned up reliably, even if an exception is thrown in the test case.